clusters/app.ci/pr-reminder/pr-reminder-config.yaml

```diff
@@ -71,3 +71,40 @@ data:
         repos:
           - openshift/baremetal-runtimecfg
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
+          - jsafrane
+          - rbednar
+        teamNames:
+          - openshift-team-storage
```

Comment:
Should be `storage` included too? https://github.com/orgs/openshift/teams/storage/members
That's what I use to get our team attention

Reply:
Most likely, adding.